visualization/app/codeCharta/ui/legendPanel/legendPanel.component.html

```diff
@@ -66,5 +66,6 @@
 	class="panel-button md-raised md-primary cc-shadow"
 	ng-click="$ctrl.toggle()"
 	ng-class="{ sideBarVisible: $ctrl._viewModel.isSideBarVisible }"
+	title="Toggle Legend-Panel"
```

Comment:
Just a thought: it would actually be possible to use `Show Panel` and `Hide Panel` instead. Not a blocker though.

Reply:
done